sdk/framework/field_data_test.go

```diff
@@ -802,6 +802,39 @@ func TestFieldDataGet(t *testing.T) {
 			"foo",
 			http.Header{},
 		},
+
+		"float type, positive with decimals, as string": {
```

Comment:
Might be also good to include a `"type float, not supplied"` test case (similar to the other cases that tests for fallback to their zero-value).

Reply:
done